internal/route/routes.go

```diff
@@ -45,16 +46,17 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 			// When I sanitize the string before splitting
 			// it doesn't work, so for now I'm just sanitizing
 			// each line
-			t = p.Sanitize(strings.TrimSpace(t))
-			todos = append(todos, t)
+			todoList.Todos = append(todoList.Todos, &model.Todo{
+				Todo: p.Sanitize(strings.TrimSpace(t)),
+			})
```

Comment:
and here we add an actual `Todo` (struct) to the actual `Todos` field. Again we're using the data we already defined and exported instead of intermediary variables and types.